openspp-project/openspp-registry
# add gender data and educational level selections

## **Why is this change needed?**
1. To not install `spp_base_demo` when `spp_farmer_registry_demo` is installed.
2. To fix the problem with the Highest Educational Level.

## **How was the change implemented?**
1. Removed `spp_base_demo` on `spp_farmer_registry_demo` depends, and add gender data xml.
2. Added the same selection values from `res_partner` to `spp_farmer`.

## **New unit tests**
```
None
```

## **Unit tests executed by the author**
```
None
```

## **How to test manually**
- Install `spp_farmer_registry_demo` on new instance or make sure `spp_base_demo` is not installed by uninstalling `spp_farmer_registry_demo` first.

Test 1
- Check if `spp_base_demo` is installed (should not be installed automatically).

Test 2
- Go to Configuration > Gender Types.
- Check if Male and Female are existing in the Gender Types.

Test 3
- Go to Group > Select or Open one.
- Check if Farmer Highest Educational Level selections is the same with Individual's Highest Educational Level.

## **Related links**
https://github.com/orgs/OpenSPP/projects/5/views/4?pane=issue&itemId=81324763
